src/Microsoft.IdentityModel.Logging/TextWriterEventListener.cs

```diff
@@ -91,7 +91,7 @@ protected override void OnEventWritten(EventWrittenEventArgs eventData)
 
             for (int i = 0; i < eventData.Payload.Count; i++)
             {
-                _streamWriter.WriteLine(eventData.Payload[i].ToString());
+                _streamWriter.WriteLine(eventData.Payload[i]!.ToString());
```

Comment:
is this safe?

Reply:
why do you think this is needed @FuPingFranco ? Are you getting a warning in the IDE?

Reply:
@keegan-caruso I believe it is safe, null checks for  eventData and Payload are checked on lines 83 and 86 respectively.
@jennyf19  Yes, I get warnings for error code CS8602-Dereference of a possibly null reference. (happens when we set nullables but it doesn't know we already performed null checks previous to using it this way)

Reply:
should be `_streamWriter.WriteLine(eventData!.Payload[i]?.ToString());`